src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
```

Comment:
better create a constant `NUMBER_OF_BALLS` for number 3 and use it, because now it's a magic number
read more about constants here - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names  (constant should have `private static final` modifiers)
```suggestion
        Ball[] balls = new Ball[NUMBER_OF_BALLS];
```